admin/includes/languages/english/orders.php

```diff
@@ -130,3 +130,5 @@
 define('TEXT_MAP_SHIPPING_ADDRESS', 'Map Shipping Address');
 define('TEXT_MAP_BILLING_ADDRESS', 'Map Billing Address');
 
+define('TEXT_EMAIL_LANGUAGE', 'order language: %s');
```

Comment:
Did you intend this to be all lowercase?

Reply:
Since it had the icon in front, yes. But I've changed it so they are capitalised.

Reply:
Okay, the first time I'd looked at this I didn't have an icon set for the 2nd language, so I didn't have anything showing beside it. And then looking at the language file which doesn't reflect that context it seemed odd that it was all lowercase, and I figured there'd be people coming along doing translation who would complain about it, so it made sense to ask! 😊